backend/app/handlers/prompt_for_image.py

```diff
@@ -0,0 +1,77 @@
+from app.lm.models import (
+    Message as ChatCompletionMessage,
+)
+import base64
+from typing import BinaryIO
+from app.models import DocumentDataExtractor
+from app.ops import tup
+from app.ops.documents import as_png
+from app.services.pdf_reader import pdf_reader
+from app.services.png_reader import png_reader
+from app.models import ContentType
+
+BASE64_IMAGE_PREFIX = "data:image/png;base64"
+     
+def fill_input_image(buffer: BinaryIO) -> dict:
+    img_bytes = buffer.getvalue()
+    base64_image=(base64.b64encode(img_bytes).decode('utf-8'))
+    return {
+        "type": "image_url",
+        "image_url": {
+            "url": f"{BASE64_IMAGE_PREFIX},{base64_image}",
+        }
+    }
+    
+#TODO rewrite with ChatCompletionMessage
+#link to the OpenAI documentation specifying that only role 'user' is used for images:
+#https://platform.openai.com/docs/guides/vision
+async def full_prompt_from_image(file: BinaryIO, document_data_extractor: DocumentDataExtractor, upload_content_type: ContentType) -> list[ChatCompletionMessage]:
+    if upload_content_type == ContentType.PDF:
+        prompt = pdf_reader.as_png(file)
+    if upload_content_type == ContentType.PNG:
+        prompt = png_reader.as_png(file)
+    
```

Comment:
else: 
    raise not ImplementedError(f'Content type {upload_content_type} not supported')